Makefile

```diff
@@ -108,4 +109,4 @@ docs.generate: ## Generate docs
 	swag init --dir cmd,internal --output docs
 
 docs.toOpenAPI: ## Convert docs to OpenAPI format
-	swagger2openapi docs/swagger.json -o docs/openapi.json
\ No newline at end of file
+	swagger2openapi docs/swagger.json -o docs/openapi.json
```

Comment:
@sakydev Can you review this PR and see what this change is? I did not change anything on this line.